Tactics/CSE.lean

```diff
@@ -125,13 +132,10 @@ structure State where
   a counter to generate new names.
   -/
   gensymCount : Nat := 1
-
-instance : ToMessageData State where
-  toMessageData s := Id.run do
-    let mut msg := m!""
-    for (k, v) in s.canon2data.toArray do
-      msg := msg ++ Format.line ++ m!"{k} := {repr v}"
-    msg
```

Comment:
Why did this instance disappear?

Reply:
@alexkeizer because I stopped using the instance, it was far too verbose; I opt for more controlled printing now.